packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -469,7 +469,9 @@ export default class Meeting extends StatelessWebexPlugin {
   resourceId: any;
   resourceUrl: string;
   selfId: string;
-  state: any;
+  state: any;  
+  isMultistream: boolean;
```

Comment:
I think you meant to do this change on a branch from "beta" branch and merge this PR into "beta" branch, not master. We don't have the isMultistream parameter on SDK master

Reply:
do you mean we have 2 branch, one named `master` for transcoding meeting and other other named `beta` for multiple stream meeting ? Further more, I should create a new pr to merge to code changes to `beta` but not `master`, correct ?